packages/usehooks-ts/src/useClickAnyWhere/useClickAnyWhere.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react'
+
+type AnyEvent = MouseEvent | TouchEvent
+type Handler = (event: AnyEvent) => void
```

Comment:
Can be simplified to 

```suggestion
type Handler = (event: MouseEvent) => void
```

Indeed, the `TouchEvent` is keyboard related and not called in the hook.